packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/ReactExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package expo.modules.kotlin
+
+import com.facebook.react.bridge.ReactContext
+import expo.modules.adapters.react.NativeModulesProxy
+
+internal fun ReactContext.getUnimoduleProxy(): NativeModulesProxy? {
```

Comment:
```suggestion
internal fun ReactContext.getNativeModulesProxy(): NativeModulesProxy? {
```

nit: maybe not call it as unimodules since we would like to deprecate the name

Reply:
It was intentional. It will be much easier to know that we should remove it in the future as it's part of the legacy init logic.